caravel/assets/visualizations/big_number.js

```diff
@@ -97,6 +102,19 @@ function bigNumberVis(slice) {
         .style('font-size', d3.min([height, width]) / 3.5)
         .attr('fill', 'white');
 
+      //Printing big number subheader text
+      if (json.subheader !== null) {
+        g.append('text')
+          .attr('x', width / 2)
+          .attr('y', y + d3.min([height, width]) / 4.5)
+          .text(json.subheader)
+          .attr('id', 'subheader_text')
```

Comment:
for future, I think moving away from using id's as much as possible is ideal, they create global JS vars so using classes is preferred. also in the case that we do need them I think camel case is preferred.